chat-plugins/mafia-data.js

```diff
@@ -83,6 +98,59 @@ exports.alignments = Object.assign(Object.create(null), {
 		],
 		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
 	},
+	mime: {
+		name: `Mime`,
+		plural: `Mimes`,
+		id: `mime`,
+		color: `#A6B`,
+		memo: [
+			`Factional Communication: If there are other Mime-aligned players, you may PM them during the game.`,
+			`Factional Roleblock: The Mimes may roleblock one player per night.`,
+			`You are aligned with the <span style="color:#A6B;font-weight:bold">Mimes</span>. You win when all players with a Mime wincon have been lynched.`,
+		],
+		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
+	},
+	arsonist: {
+		name: `Arsonist`,
+		plural: `Arsonists`,
+		id: `arsonist`,
+		color: `#E82`,
+		memo: [
+			`Factional Communication: If there are other Arsonist-aligned players, you may PM them during the game.`,
+			`Factional Actions: The Arsonists may together use one factional action a night.`,
+			`Factional Prime: The Arsonists prime someone by dousing them with gasoline.`,
+			`Factional Ignite: The Arsonists light all previously primed players on fire, killing them.`,
+			`You are aligned with the <span style="color:#E82;font-weight:bold">Arsonists</span>. You win when all players without an Arsonist wincon are eliminated and at least one Arsonist-aligned player is still alive (or nothing can prevent the same).`,
+		],
+		image: `<img width="75" height="75" src="//play.pokemonshowdown.com/fx/mafia-villager.png"/>`,
+	},
+	pygmee: {
+		name: `Pygmee`,
+		plural: `Pygmees`,
+		id: `pygmee`,
+		color: `#640`,
+		memo: [
+			`Factional Knowledge: If there are other Replicant-aligned players, you know who they are.`,
```

Comment:
ahem

Reply:
Actually went through all OSes to see if there were non-Mafia copied alignments. As I commented in the file, I am alright with adding Terrorist / Mermaids as well, but I'd like to get a re-review for those themes first. I've talked with the creator from Mermaids and that one is going to be revamped first anyway.